react/BaseContent.tsx

```diff
@@ -15,15 +15,20 @@ interface Props {
   finishShoppingButtonLink: string
 }
 
-const CSS_HANDLES = ['minicartContent', 'minicartFooter'] as const
+const CSS_HANDLES = [
+  'minicartContentContainer',
+  'minicartProductListContainer',
+  'minicartTitle',
+  'minicartFooter',
+] as const
 
 const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
   const { orderForm, loading }: OrderFormContext = useOrderForm()
   const handles = useCssHandles(CSS_HANDLES)
   const { variation } = useMinicartState()
   const { url: checkoutUrl } = useCheckoutURL()
 
-  const minicartContentClasses = `${handles.minicartContent} ${
+  const minicartContentClasses = `${handles.minicartContentContainer} ${
```

Comment:
Isn't that a breaking change?

Reply:
Not in this case, since nobody is using `minicart.v2` yet